www/horl13/sp2/js/script.js

```diff
@@ -0,0 +1,396 @@
+var emailclient = {

```

Comment:
tohle taky dovnitr `$(document).ready(function() {})`, to same plati pro ty funkce dole